toolchain/ext.bzl

```diff
@@ -1,6 +1,19 @@
 load("@hermetic_cc_toolchain//toolchain:defs.bzl", zig_toolchains = "toolchains")
 
+_HOSTS = (
```

Comment:
Let's declare a list of tuples like rules_go does, so we don't need to split it later: 

https://github.com/bazel-contrib/rules_go/blob/87e66bb70858570ac668f95662d6070417d5a416/go/private/extensions.bzl#L99-L106